abuseipdb/help.md

```diff
@@ -289,6 +289,7 @@ There's a rate limit on the free API service. The following error messags `429 C
 
 # Version History
 
+* 5.0.2 - Hub styling update
```

Comment:
We can keep this at 5.0.1 since we didn't release the last one

Reply:
Same for others that have last changelog line of `New spec and help.md format for the Hub`